client/css/ImageGallery.scss

```diff
@@ -1,40 +1,38 @@
 .imageGalleryContainer {
-  display: flex;
-  width: 100%;
   align-items: center;
   background-color: rgb(128, 128, 128);
-}
-
-.mainImageContainer {
   display: flex;
-  height: 100%;
-  width: 100%;
-  justify-content: space-around;
-  align-items: center;
-  order: 3;
-  background-color: rgb(128, 128, 128);
-};
-
+  flex: 2;
+  padding: 30px;
+
+  &__navigation {
+    align-items: center;
+    display: flex;
+    flex: 1;
+    justify-content: space-between;
+    padding-left: 30px;
+  }
+}
 
 .focusedImage {
   background-position: center;
   background-repeat: no-repeat;
-  background-size: 100%;
-  height: 99%;
-  width: 100%;
+  background-size: cover;
   cursor: zoom-in;
+  display: flex;
+  flex-direction: row;
 }
 
 .focusedImageExpand {
   background-repeat: no-repeat;
   background-size: 100%;
   height: 100%;
   width: 100%;
-}
 
-.focusedImageExpand:hover {
-  background-size: 175%;
-  cursor: crosshair;
+  &:hover {
+    background-size: 175%;
```

Comment:
I noticed upon tinkering with the image zoom, the image spazzes out when the mouse crosses the horizontal midpoint of the image. I might have to try a different implementation of the zoom feature if given enough time.